app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -2386,4 +2388,30 @@ public function buildUrl($route, $routeParams = [], $absolute = true, $clickthro
 
         return $url;
     }
+
+    /**
+     * @return array<int, array<string, int|string>>
+     *
+     * @throws Exception
+     */
+    public function getEmailDayStats(Lead $lead): array
```

Comment:
Do not see the reason to create these two public  methods instead of the call methods directly from repo